src/components/CodeExample/index.css

```diff
@@ -0,0 +1,42 @@
+@import "../custom-properties.css";
+
+/** @define CodeExample; */
+
+.CodeExample-button {
+  background: transparent;
+  border: none;
+  display: inline-block;
+  font: 600 var(--type-2) proxima-nova;
+  margin: 0 1px;
+  padding: .25rem 2.5rem;
+  text-align: center;
+  text-transform: capitalize;
+  vertical-align: middle;
+
+  &:hover, &.selected {
+    background-image: linear-gradient(var(--black), color-mod(var(--black) tint(30%)));
+    border: 1px solid #000;
+    border-radius: 5px;
+    margin: 0;
+    color: white;
+  }
+}
+
+.CodeExample-buttonGroup {
+  background-image: linear-gradient(var(--white), color-mod(var(--black) tint(80%)));
+  border: 1px solid var(--black);
+  border-radius: 5px;
+  display: -ms-inline-flexbox;
```

Comment:
@software-project even with the weird syntax, IE flexbox has a large amount of bugs. Is it worth using inline-block, floats or vertical layout as a fallback instead? https://caniuse.com/#feat=flexbox

Reply:
@derekjohnson that's a solution from twitter bootstrap if I remember correctly. Feel free to make changes as you feel. I'm not sure I followed BEM and post-css best practices.